### PR TITLE
filtering disallowed

### PR DESCRIPTION
**Pull request by sergioloporto**

### CODE REVIEW

**Review by maciekz1996 — CHANGES_REQUESTED** [review]
Codewise it looks great 👍 However when the alert appears after typing for example `admin1:` after dismissing the alert the search query is still applied resulting in `No records to display` message. I think it would be better from a user's experience perspective to prevent search query from triggering. Also I saw that some integration tests are failing

File: verification/curator-service/ui/src/components/FiltersModal.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useDebugValue } from 'react';
```
Comment (maciekz1996) [inline verification/curator-service/ui/src/components/FiltersModal.tsx:1]: `useDebugValue` should be removed as it is not used anywhere

**Review by maciekz1996 — CHANGES_REQUESTED** [review]
Couple of things that I noticed except the ones pointed out in the comments in the code: 
* There should be a margin added on the left and on the right of Alert so that it is aligned properly with the rest of Modal
* `modalAlert` variable should be set to false when the user submits filters form to not show the alert second time filters modal is opened 
* Tests should be fixed

File: verification/curator-service/ui/src/components/SearchBar.tsx [hunk 2]
```diff
@@ -133,6 +133,17 @@ export default function SearchBar({
...
     };
 
+    const disallowFilteringInSearchBar = (e: any) => {
+        e.preventDefault();
+        setIsUserTyping(false);
+        setModalAlert(true);
+        setFiltersModalOpen(true)
+    };
+
+    function handleSetModalAlert(shouldTheAlertStillBeOpen: any) {
```
[the remark this answers is not in the document]
Comment (sergioloporto) [reply verification/curator-service/ui/src/components/SearchBar.tsx]: fixed

File: verification/curator-service/ui/src/components/SearchBar.tsx [hunk 3]
```diff
@@ -133,6 +133,17 @@ export default function SearchBar({
         }
     };
 
+    const disallowFilteringInSearchBar = (e: any) => {
```
[the remark this answers is not in the document]
Comment (sergioloporto) [reply verification/curator-service/ui/src/components/SearchBar.tsx]: fixed

File: verification/curator-service/ui/src/components/FiltersModal.tsx (hunk 1 above)
Comment (sergioloporto) [reply verification/curator-service/ui/src/components/FiltersModal.tsx]: fixed

**Review by maciekz1996 — APPROVED** [review]
Works and looks great, good job 💯  After resolving merge conflict it will be ready for merging

**Review by maciekz1996 — APPROVED** [review]

**Review by abhidg — APPROVED** [review]
Looks good, thanks!

**Pull request merged**